src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Usage/DetectPLINQNopsTest.cs

```diff
@@ -92,6 +80,24 @@ class TypeName
 }", VerifyCS.Diagnostic(DetectPLINQNopsAnalyzer.DefaultRule).WithLocation(0).WithArguments("enumerable.AsParallel()"));
         }
 
+        [Fact]
+        public async Task VB_AsParallelAtEnd_SingleDiagnostic()
+        {
+            await VerifyVB.VerifyAnalyzerAsync(@"
+Imports System
+Imports System.Collections.Generic
+Imports System.Linq
+
+Namespace ConsoleApplication1
+    Class TypeName
+        Public Sub Test()
+            For Each s In {|#0:Enumerable.Range(0, 1).Select(Function(x) x * 2).AsParallel()|}.ToList()
```

Comment:
Add test where you call AsParallel without parentheses.

Reply:
Just checked, it also resolves to an IInvocationOperation.

Reply:
@Mrnikbobjeff Problems may occur when the codefix for VB is implemented. So the test is valuable IMO.
This is common scenario that fails. While semantically it resolves to an invocation, it doesn't syntactically.